### PR TITLE
refine game activity

### PR DESCRIPTION
**Pull request by bofeng-song**
Re: #  https://github.com/cocos/cocos-engine/issues/14627

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: native/cocos/platform/android/AndroidApp.h [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+    /** The current configuration the app is running in. */
+    AConfiguration* config;
+
+    /**
+     * Pointer to the Asset Manager instance for the application.  The
+     * application uses this to access binary assets bundled inside its own .apk
+     * file.
+     */
+    AAssetManager* assetManager;
```
Comment (dumganhar) [inline native/cocos/platform/android/AndroidApp.h]: Global variables were not initialized.

File: native/cocos/platform/android/AndroidApp.h [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        setSurface(nullptr);
+    }
+
+    inline void resize(uint32_t width, uint32_t height) { _width = width; _height = height;}
+    void setSurface(jobject surface);
+
+    inline ANativeWindow *getNativeWindow() const { return _nativeWindow; }
+    inline int getWindowId() const { return _windowId; }
+    inline int getWidth() const { return _width; }
+    inline int getHeight() const { return _height; }
```
Comment (dumganhar) [inline native/cocos/platform/android/AndroidApp.h]: The function return types is different with where they defined.

File: native/cocos/platform/android/AndroidApp.h [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    /** @cond INTERNAL */
+
+    pthread_mutex_t mutex;
+    pthread_cond_t cond;
+
+    pthread_t thread;
+
+    int running;
+    int destroyed;
+    ARect pendingContentRect;
```
Comment (dumganhar) [inline native/cocos/platform/android/AndroidApp.h]: Do these variables all need to be public?

Comment (bofeng-song) [reply native/cocos/platform/android/AndroidApp.h]: pendingContentRect can be removed temperary, others need to be declare as global.

**Pull request closed without merging**